### PR TITLE
Added ffnvcodec-headers

### PR DESCRIPTION
**Pull request by hmaarrfk**
@sdvillal this is your recipe. Did you want to be a maintainer here?

### CODE REVIEW

**Comment by hmaarrfk** [comment]
I updated the version, but maybe we can just make a few releases and let the downstream packages pin if they want to?

**Comment by sdvillal** [comment]
Yeah, I can be a maintainer too. Beware the minimum driver version. IIRC now ffmpeg includes dynamic checks and refuses to initialize the nvidia codecs if the system does not comply. In the past it did not, and that proved problematic to us - essentially it was able to hang the whole GPU...

If it is safe, go ahead. If it is not, what we do is to keep a separated nvidia build of our ffmpeg packages.

**Comment by hmaarrfk** [comment]
whoa crazy, alright, I'll have to check. Hopefully my workstation has an older version of the driver and I could test it on that.

Is this required for windows to use nvidia encoding for ffmpeg too?

**Comment by sdvillal** [comment]
I do not know about win. I think we do not build ffmpeg ourselves there anyway.

**Comment by hmaarrfk** [comment]
Yeah, i might be tempted to reboot after I get dinner. Thanks for your help!

I noticed that you are a maintner on ffmpeg. I'm hoping we can get this passed through conda-forge's review (should be pretty simple compared to https://github.com/conda-forge/staged-recipes/pull/6965) and we can think of a solution that works for ffmpeg + linux without crashing everybody's computers.

**Comment by hmaarrfk** [comment]
Well i added windows support, then half way through learning how to use windows to build stuff, i realized that ffmpeg on windows is compiled  with literally everything it can. It already uses 


```
 DEV.L. hevc                 H.265 / HEVC (High Efficiency Video Coding) (decoders: hevc hevc_qsv hevc_cuvid ) (encoders: libx265 nvenc_hevc hevc_amf hevc_nvenc hevc_qsv )
```

```
 DEV.LS h264                 H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10 (decoders: h264 h264_qsv h264_cuvid ) (encoders: libx264 libx264rgb h264_amf h264_nvenc h264_qsv nvenc nvenc_h264 )
```

**Comment by hmaarrfk** [comment]
@conda-forge/staged-recipes friendly ping for a review.

File: recipes/ffnvcodec-headers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/FFmpeg/nv-codec-headers/releases/download/n{{ version }}/nv-codec-headers-{{ version }}.tar.gz'
+  sha256: 9a567b45ece1930e65803411f12736d8925f0d1adbc31e962cb5c441969940f0
+
+build:
+  number: 1
```
Comment (ocefpaf) [inline recipes/ffnvcodec-headers/meta.yaml]: We usually start from `0`. (Not a big deal though.)

File: recipes/ffnvcodec-headers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  skip: True  # [osx]
+  script: make PREFIX={{ PREFIX }} install
+
+requirements:
+  build:
+    # git provides the sed command
+    - git   # [win]
```
Comment (ocefpaf) [inline recipes/ffnvcodec-headers/meta.yaml]: I wonder if we can use one of the msys smaller packages to get `sed`...

Comment (hmaarrfk) [reply recipes/ffnvcodec-headers/meta.yaml]: That’s what somebody had suggested. I tried one of them, but it complained that /tmp wasn’t available. I can iterate on that separately though. Thanks!

File: recipes/ffnvcodec-headers/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipes/ffnvcodec-headers/meta.yaml]: Dam. Sorry :/

File: recipes/ffnvcodec-headers/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/ffnvcodec-headers/meta.yaml]: That is not important. I was just wondering if that is possible. (But if you find out please let me know so I can use it :smile:)

**Pull request merged**